### PR TITLE
Git ignore

### PR DESCRIPTION
**Pull request by JamesRichards07**
Completed all current GitHub actions under the git_ignore branch. Open to suggestions and comments before merging into main.

### CODE REVIEW

**Comment by electricjones** [comment]
This is meant to resolve #1  and #2

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
-vendor
-./composer.lock
-./composer.phar
-data
+vendor/
+composer.lock
+composer.phar/
```
Comment (electricjones) [inline .gitignore]: I don't think composer.phar is a directory. You can remove the trailing /

**Comment by electricjones** [comment]
This looks great. I made one comment about the .gitignore file.
However, it seems that the `.idea` folder and `composer.phar` are actually still in the repository. I see you added them to `.gitignore`. Did you run the 

```
git rm --cached -r .idea
git rm --cached composer.phar
```
Commands? And then you have to make a commit and push them.

**Comment by electricjones** [comment]
Other than the couple nitpicks I mentioned in this review, this looks great! Feel free to merge it anytime. I'll go ahead and approve it.

When you merge, you have three choices:
1. Create a Merge Commit -- which will merge the three or four commits you have in this branch just as it is
2. Squash and Merge -- Will squash all of these commits into one commit named whatever the PR is named (or you can change it). So, in `master` it looks like this was all one commit.
3. Rebase and Merge -- don't do this. It rewrites history and is weird

You can choose either 1 or 2. There really isn't a best practice. The reason you might choose 2 is if this PR has a bunch of commits like "I broke it" and "again". Which is pretty common. You don't have that, so it really doesn't matter.

Great job.

**Review by electricjones — APPROVED** [review]
Approved.

**Comment by JamesRichards07** [comment]
So when I run 'git rm --cached composer.phar' I get: "fatal: pathspec 'composer.phar' did not match any files". Not sure why those files are still showing up.

File: .gitignore (hunk 1 above)
Comment (JamesRichards07) [reply .gitignore]: Yeah, it's an archive which has directories in it. Slash removed.

**Pull request merged**